zerver/openapi/zulip.yaml

```diff
@@ -6189,7 +6189,10 @@ paths:
             subscribed to appropriate streams or use a shared history
             search narrow with this endpoint.
 
-            **Changes**: In Zulip 9.0 (feature level 250), narrows gained support
+            **Changes**: In Zulip 9.0 (feature level 254), added new `is:followed`
+            filter, matching messages from followed topics.
```

Comment:
I'd revise to something like: `filter, which returns messages in a user's [followed topics](help-link).`

It's always good to link to the help center article that explains a feature.